web-ng/src/app/components/map/map.component.ts

```diff
@@ -207,16 +229,34 @@ export class MapComponent implements AfterViewInit, OnDestroy {
     };
     const marker = new google.maps.Marker(options);
     marker.addListener('click', () => {
-      this.selectMarker(marker);
+      if (this.disableMapClicks) {
+        return;
+      }
       this.navigationService.selectFeature(feature.id);
     });
+    marker.addListener('dragstart', (event: google.maps.MouseEvent) => {
```

Comment:
Can we break these listeners out into their own functions for readability?

Reply:
Done.